karma.conf.js

```diff
@@ -41,14 +40,13 @@ settings = {
   },
   reporters: ['dots', 'coverage'],
   files: files,
-  frameworks: ['mocha', 'chai', 'sinon', 'browserify', 'nunjucks'],
+  frameworks: ['mocha', 'chai', 'sinon', 'browserify'],
```

Comment:
Sort of sad to see this go, but per our conversation I agree that we should accomplish the dream of testing the full FE functionality from another direction.